src/ui/screens/VerificationRequirementScreen.tsx

```diff
@@ -0,0 +1,112 @@
+import Grid from '@material-ui/core/Grid';
+import * as React from "react";
+
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+import I18n from '../utils/I18n';
+import { IAgent } from '../interfaces/AgentInterfaces';
+import KivaAgent from '../agents/KivaAgent';
+import LocalAgent from '../agents/LocalAgent';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select';
+import {VerificationRequirementProps, VerificationRequirementState} from '../interfaces/VerificationRequirementProps';
+import {flowController} from "../KernelContainer";
+import auth from '../utils/AuthService';
+import _ from 'lodash';
+
+import '../css/Common.css';
```

Comment:
You should be able to import using the `scss` extension. And posible import Common earlier in the app so that is only added once.